src/app/clusters/fan-control-server/fan-control-server.cpp

```diff
@@ -228,10 +229,11 @@ MatterFanControlClusterServerPreAttributeChangedCallback(const ConcreteAttribute
     case WindSupport::Id: {
         if (SupportsWind(attributePath.mEndpointId))
         {
-            uint8_t windSupport;
-            EmberAfStatus status = WindSupport::Get(attributePath.mEndpointId, &windSupport);
+            chip::BitMask<WindBitmap> windSupport;
+            EmberAfStatus status   = WindSupport::Get(attributePath.mEndpointId, &windSupport);
+            uint8_t rawWindSupport = windSupport.Raw();
```

Comment:
See comments above for RockSupport.

That said (pre-existing, please file an issue) this code makes no sense.  It's constraining what you can set the "WindSupport" attribute to based on the _current_ value of that attribute.  The case should be `WindSetting::Id`, no?